deploy.sh

```diff
@@ -293,7 +297,27 @@ check_service_status() {
         echo "$servicestatus"
 }
 
+deploy_email_template() {
+        
+  echo "Deploying email template"
+
+  template=`cat ./emails-dist/template.html`
+  jq -n --arg template "$template" \
+   '{"active":1,"html_content":$template}' > request-data.txt
```

Comment:
@gondzo Should we assume that it is always an active template version? Although, I don't see any use case of having id of an inactive template in the config, but I would like to avoid this just in case we have activated a different version in sendgrid than what we have configured in circle ci.